### PR TITLE
fix(dropdown): fix dropdown select all checked by default

### LINKED ISSUE #637: Dropdown select all checked by default if it's set to multiple
**Describe the bug**
When the dropdown component is set to multiple by default, the select all checkbox is checked even if the value is empty.

**To Reproduce**
1. Create and define a dropdown element with multiple property
2. click on the component
3. see select all checked

**Expected behavior**
select all should not be checked if value is empty

### PR DESCRIPTION
**Pull request by kherP**
fix #637

### CODE REVIEW

File: lib/src/Dropdown/Dropdown.tsx [hunk 1]
```diff
@@ -71,41 +71,39 @@ export const Dropdown: React.FC<DropdownProps> = React.forwardRef(
...
 
-        const isAllSelected = (): boolean => {
-            return Array.from(selectRef.current.options).every((_, i) => {
-                const option: HTMLOptionElement = selectRef.current.options.item(i);
+        const isAllSelected = React.useCallback((): boolean => {
+            return selectRefOptions.every((option: HTMLOptionElement) => {
                 return option.disabled ? true : option.selected;
             });
-        };
+        }, [selectRefOptions]);
```
Comment (eweseong) [inline lib/src/Dropdown/Dropdown.tsx:106]: does all selection includes options that are disabled and not selected?

Comment (kherP) [reply lib/src/Dropdown/Dropdown.tsx:106]: yes, selectRefOptions returns every option within the select node

**Review by eweseong — APPROVED** [review]

**Review by mario-subo — APPROVED** [review]

**Pull request merged**